Wox.Plugin/SharedCommands/FilesFolders.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.IO;
+
+namespace Wox.Plugin.SharedCommands
+{
+    public static class FilesFolders
```

Comment:
can we maybe do this a non static class with an interface?

Reply:
I think these are OK to remain static because they are straightforward utility commands that don't require object reference and common enough to be used by all plugins. In the future if they do become plugin specific then they can be easily converted, then again if they are, they should live locally with the plugin's directory anyway.

Reply:
totally agree with that. I think it is always the problem later on if you want to try and write tests for it the static part doesn't let you mock the file system, on the other hand I really doubt we ever get to there :)